### PR TITLE
Refs [TASK][35415] Create UI Match Events Screen

### PR DESCRIPTION
**Pull request by phucsaker99**
## Related Tickets
- [#Task 35415: Create UI Match Events Screen](https://edu-redmine.sun-asterisk.vn/issues/35415)
## WHAT
- Create RecylerView list notification by date by date
- Create datePicker
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/example/soccert/ui/adapter/HomeViewPagerAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class HomeViewPagerAdapter(
+    fragmentManager: FragmentManager
+) : FragmentStatePagerAdapter(fragmentManager, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
+
+    private val fragments = mutableListOf<Fragment>()
+    private val titles = mutableListOf<String>()
+    override fun getItem(position: Int) = fragments[position]
+
+    override fun getCount() = fragments.size
+    override fun getPageTitle(position: Int) = titles[position]
```
Comment (thonc-2135) [inline app/src/main/java/com/example/soccert/ui/adapter/HomeViewPagerAdapter.kt]: tách private vs overide , còn dưới thì k cần tách hoặc tách 1 lượt

File: app/src/main/java/com/example/soccert/ui/adapter/LeagueAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import com.example.soccert.R
+import com.example.soccert.base.BindDataAdapter
+import com.example.soccert.data.model.Competition
+import com.example.soccert.databinding.ItemLeagueBinding
+import com.example.soccert.databinding.ItemLeagueSelectedBinding
+
+class LeagueAdapter(
+    context: Context
+) : ArrayAdapter<Competition>(context, 0), BindDataAdapter<Competition> {
+
```
Comment (thonc-2135) [inline app/src/main/java/com/example/soccert/ui/adapter/LeagueAdapter.kt:19]: sao dùng arrayadapter thế

**Review by datdt-1935 — APPROVED** [review]

**Review by huytq-2422 — APPROVED** [review]

File: app/src/main/java/com/example/soccert/ui/adapter/LeagueAdapter.kt (hunk 2 above)
Comment (phucsaker99) [reply app/src/main/java/com/example/soccert/ui/adapter/LeagueAdapter.kt:19]: e dùng spinner ạ

File: app/src/main/java/com/example/soccert/ui/adapter/HomeViewPagerAdapter.kt (hunk 1 above)
Comment (phucsaker99) [reply app/src/main/java/com/example/soccert/ui/adapter/HomeViewPagerAdapter.kt]: vâng a

**Review by thonc-2135 — APPROVED** [review]

**Pull request merged**